ios-target-app/Onboarding/Views/SignInViewController.swift

```diff
@@ -78,18 +83,33 @@ class SignInViewController: UIViewController {
         setViews()
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: true)
```

Comment:
the animated flag should follow the appearance animated flag to avoid weird visual effects.